EndlessClient/Network/UnknownEntitiesRequester.cs

```diff
@@ -0,0 +1,104 @@
+﻿using EndlessClient.GameExecution;
+using EOLib.Domain.Map;
+using EOLib.Net;
+using EOLib.Net.Communication;
+using Microsoft.Xna.Framework;
+using System;
+
+namespace EndlessClient.Network
+{
+    public class UnknownEntitiesRequester : GameComponent
```

Comment:
This is exactly the pattern I would have implemented for this type of thing, nice job